Sources/SentryCrash/Recording/SentryCrash.h

```diff
@@ -261,38 +261,6 @@ typedef enum {
  */
 - (void)deleteReportWithID:(NSNumber *)reportID;
 
-/** Report a custom, user defined exception.
```

Comment:
This code had an analyze error in the `.m` file. This method is not used anymore. Instead of fixing the error, I removed the code.